loops_lab.js

```diff
@@ -0,0 +1,197 @@
+//1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+let num = 15
+while (num <= 15 && num > 0){
+  console.log(num);
+  num -=1;
+}
+
+for(let num = 15; num <= 15 && num > 0; num -=1){
+  console.log(num);
+}
+
+// 2. Write a `while` loop and a `for` loop that takes the variable "max", and iterates over all numbers from 0 to "max".
+// For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
+let max = 0;
+while (max <= 88){
+  if(max % 2 === 0){
+    console.log(max + " is even.");
+  }
+if(max % 2 !== 0){
+    console.log(max + " is odd.");
+  }
+max++;
+}
+
+for(let max = 0; max <= 88; max ++){
+  if(max % 2 === 0){
+    console.log(max + " is even.");
+  }
+  if(max % 2 !== 0){
+    console.log(max + " is odd.");
+  }
+}
+
+// 3. Write a `while` loop and a `for` loop that takes the variable "x" and iterates over all numbers from 0 to "x".
+// For each iteration of the loop, it will multiply the number of "x" by 9 and log the result (e.g. "2 * 9 = 18").
+let x = 0;
```

Comment:
Would this code still work even if you change x to 1? It is important to write a code that can be adopted to differing perimeters and numbers.